tools/SourceKit/tools/sourcekitd-test/sourcekitd-test.cpp

```diff
@@ -416,6 +416,46 @@ static bool readPopularAPIList(StringRef filename,
   return false;
 }
 
+/// Read '-req-opts' for syntactic macro expansion request and apply it to 'req'
+/// object.
+/// The format of the argument is '-req-opts={line}:{column}:{path}'
+/// where {path} is a path to a JSON file that has macro roles and definition.
+/// {line} and {column} is resolved to 'offset' using \p inputBuf .
+static bool setSyntacticMacroExpansions(sourcekitd_object_t req,
+                                        TestOptions &Opts,
+                                        llvm::MemoryBuffer *inputBuf) {
```

Comment:
Nit: This function really mixes lowercase and uppercase variable names. It would be nice if we were consistent at least within functions.